sos/report/plugins/networkmanager.py

```diff
@@ -31,6 +36,10 @@ def setup(self):
 
         self.add_journal(units="NetworkManager")
 
+        nm_ver = '.'.join(nm_pkg['version'])
+        if parse_version(nm_ver) >= parse_version('1.2'):
```

Comment:
You don't really need to check for the version either. If the command doesn't work, then that's ok, it will not create the file

Reply:
Understood, too. I removed the version check, thanks.